akka-typed-tests/src/test/scala/akka/typed/DeferredSpec.scala

```diff
@@ -79,14 +79,36 @@ class DeferredSpec extends TypedSpec {
 
     def `must stop when exception from factory`(): Unit = {
```

Comment:
No, it never was valid since it didn’t verify that the actor has stopped. And it obviously was not enough to catch the missing behavior added by this PR.